### ISSUE TITLE
Allow setting cache size to < 1GB

### ISSUE THREAD

**Issue by debdattabasu** [mongodb-partners/mongo-rocks#132]
First of all, thank you for an awesome library. :)

I am running MongoRocks on memory constrained environments, where IMO RocksDB performs better than WiredTiger, which tends to freeze and hang frequently. However, I can not set the cache size to let's say 0.256GB like I can do in WiredTiger. 

Is there a possibility of making this happen? 

Regards
Debdatta Basu

**Comment by igorcanadi** [comment]
That's correct, looks like cacheGB is an integer. We'd just need to port something like this over: https://github.com/mongodb/mongo/commit/b6397d9014bb35490d8885675b9cfec2e06d9b58#diff-6a2e67ff05825f6d310e2938f3cfdd30

Interested in giving it a shot?
